packages/react-core/src/components/Switch/Switch.tsx

```diff
@@ -12,10 +12,12 @@ export interface SwitchProps
   id?: string;
   /** Additional classes added to the switch */
   className?: string;
-  /** Text value for the visible label when on */
+  /** Text value for the visible label */
   label?: React.ReactNode;
-  /** Text value for the visible label when off */
-  labelOff?: React.ReactNode;
+  /** Adds an accessible name to the switch when the label prop is not passed, and must describe the isChecked="true" state. */
+  'aria-label'?: string;
+  /** Adds an accessible name to the switch via a list of referenced id's. The computed accessible name must describe the isChecked="true" state. */
```

Comment:
Not necessarily a suggestion, but I wonder if someone will think it needs to be a list of id's (versus being able to use a single id) the way it's worded currently?

```suggestion
  /** Adds an accessible name to the switch via referenced id(s). The computed accessible name must describe the isChecked="true" state. */
```